wiki/Tournaments/GBC/GBC_2024_Spring/en.md

```diff
@@ -0,0 +1,221 @@
+6. [Tournament staff members](/wiki/Tournaments/Official_support#staff) must not participate as players in the tournament.
+7. Organisers may force to promote some players that are considered too strong in their levels.
+8. Organisers reserve the right to decline restricted players from participating in the tournament.
+9. It is allowed for players to request to promote their level. They need to send some result screenshotsof next level to the organiser for getting premission.
+
+### Win conditions
+
+1. In the Regular stage, players need to get higher placement in higher level for higher points. The points that can be earned are determined by the placement in every round. The placement according to the average rank for all individual beatmaps in a round.
+2. We have a base points that depends on the placement per round, and a multiplier that depends on the level of players. Here is the calculation method of points.
+
+| Placement | Base points |
+| :-: | --: |
+| 1st place | 20 |
+| 2nd place | 19 |
+| 3rd place | 18 |
+| 4th place | 17 |
+| Top 15% | 16 |
+| Top 22% | 15 |
+| Top 30% | 14 |
+| Top 40% | 13 |
+| Top 50% | 12 |
+| Top 60% | 10 |
+| Top 70% | 8 |
+| Top 80% | 6 |
+| Top 90% | 4 |
+| Top 100% | 2 |
+
+| Level | Multiplier (x is base points) |
+| :-: | --: |
+| Level B | x |
+| Level A | 1.5x + 12 |
+| Level S | 2.3x + 30 |
+| Level EX | 3.4x + 57 |
+
+3. The final qualifier ranking depends on the total amount of points from all 4 rounds. The top 24 players will be able to join the playoff stage.
+4. The top 8 will play in the winners' bracket, while the top 9–24 will play in the losers' bracket.
+5. In case of a tie in the rankings, the player with a lower sum of ranks in every round will take the higher rank.
+6. In the playoff stage, players need to win 6 beatmaps in order to win a match during Quarterfinals and Semifinals, and 7 beatmaps in the Finals and Grand Finals.
+
+### Match procedures
+
+1. A referee will create a multiplayer room 5 minutes before the scheduled match time. Players must join the lobby in that period.
+   - The room settings are `Game mode: osu!mania`, `Team mode: Head-to-Head`, and `Score mode: ScoreV2`. The room name must follow the pattern of `GBC 2024 Spr`.
+2. Possible mod choices for the FreeMod bracket are **No Fail**, **Hidden**, **Fade In**, **Flashlight**, and **Mirror**.
+3. The referees will select the beatmaps in order to play the match. If a player encounters a technical problem during a match, players may request a rematch from the referee after all the beatmaps have been selected.
+   - "Lag spikes" will not be considered as a valid reason to rematch a beatmap.
+   - The rematch should happen after completing scores for the other beatmaps in the mappool.
+   - For the behavior that quit room and request rematch after the map start, it will only be allowed once in a match, and also may vetoed by referee.
```

Comment:
i don't understand what this means at all (and dare i say it looks machine-translated)

Reply:
Not machine-translated but really hard to explain.
According to the rules above, it is allowed to request for a rematch for the technical issue. But someone will directly quit the room or afk after a miss and claim they have technical issue, so in order to prevent such abuse behavior more clearly, I added this limit.

Reply:
you already have `This rule should not to be abused. Referees may veto a rematch request if they find that this is the case.`, so you don't even need this rule. you could just call such behaviour foul play and reject the rematch

Reply:
Such players will claim we abuse "the rule of abuse", it is just a realistic experience that we'd better clarify something that spy the rule boundaries, or they do will do that and say you didn't prohibit such behavior in rule.

Reply:
to clarify, this is just a limit for rematching, right?

```suggestion
   - This is only allowed once per match and may also be vetoed by the referee.
```

Reply:
Yes it is. But the reason why I emphasize "quit room" is, there are still some players will directly disappear between maps, like they stuck in the score page and can't quit the room or join again, so such thing we don't have limit of rematch. Such technical issue is really common (especially in CN, we'll call it as "corpse"), so I have to clarify it more clearly.

Reply:
```suggestion
   - Quitting the room and requesting a rematch after the map starts will only be allowed once in a match, and may also be vetoed by a referee.
```